pkg/nginx/nginx.go

```diff
@@ -90,7 +90,13 @@ server {
 	server_name	"";
 	root	{{.Root}};
 
+	{{if .ServesStaticFiles}}
+	location / {
+		try_files $uri /{{.FrontControllerScript}}$uri;
```

Comment:
@kennethye1 

Sorry to bother you like this... but I am currently having an issue with GCP AppEngine Flex and PHP 8.3 where my query parameters in PHP are cut off...

I have enabled `NGINX_SERVES_STATIC_FILES: true`

Serving the static files works... but other requests to PHP are missing the query params...

Shouldn't the above line not be something like this:

```
location / {
		try_files $uri /{{.FrontControllerScript}}$is_args$args;
}
```

Cheers
Michael

Reply:
Hi Michael,

You are right, sorry for missing this. I sent another pull request to fix this: https://github.com/GoogleCloudPlatform/buildpacks/pull/418 @kennethye1 

In the meantime, if you are in a rush to fix your app, you can use this [workaround](https://gist.github.com/sonnysasaka/f3fab0c269891a3654d935dff14b0d63) first instead of the `NGINX_SERVES_STATIC_FILES`.